### PR TITLE
Add Form part

### PR DESCRIPTION
**Pull request by tusinahmed**
In this Pull my coding partner and me together doing the form and footer part. We use HTML and CSS.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -320,7 +320,54 @@ <h1>Skills</h1>
...
       <!-- Skill section end  -->
 
+
+      <!-- Form part start -->
+      <div class="dayall"> 
+      <section class="full-form-part">
+        <h1 class="form-into-text">I'm always <br> interested in <br> hearing about <br> new projects, so <br> if you'd like to <br> chat please get <br> in touch. </h1>
+
+        <div class="contact" >
+          <Form action="" method="post" class="inside-form-full" >
```
Comment (christianbiring1) [inline index.html:330]: - [ ] As mentioned in the project requirement, Kinldy make sure you use the formspree service to collect the user inputs on your page.
Hint: You may consider checking that how to use the formspree [here](https://www.youtube.com/results?search_query=formspree+contact+form)

File: index.html [hunk 2]
```diff
@@ -320,7 +320,54 @@ <h1>Skills</h1>
...
+            <div class="info">
+              <label for="fullname"></label>
+              <input type="text" name="fullname" class="fullname" placeholder="Full name" required maxlength="30">
+              <label for="email"></label>
+              <input type="email" id="email" class="emailAdress" placeholder="Email address">
+            </div>
+            <div class="message">
+              <label for="message"></label>
+              <input type="text" class="message" name="message" placeholder="Het I'd like to connect to" maxlength="500"> 
+            </div>
```
Comment (christianbiring1) [inline index.html:340]: - [ ] Also make sure the user cannot submit the form if at least one field is empty. You may consider achieving this by adding the `required` attribute to all your input fields

**Review by christianbiring1 — COMMENTED** [review]
Hi @tusinahmed @mariegrace31 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title`.

## Required Changes ♻️
- [ ] Kindly make sure you use the Formspree service correctly to collect the user inputs on your page. you may consider following this tutorial to get an idea of what you are supposed to do.

_Check the comments under the review._

## Optional suggestions
- [ ] Also please always make sure you provide a descriptive summary of your PR telling what changes have been introduced.
Hint: You may consider saying something like:
`Add the contact form section`, `validate the form inputs`, `use the formspree service to collect the user inputs..`
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] You may consider fixing the style of your page, as you may notice there's a small scroll on the left-right. make sure the width of your age fits 100% of the screen size.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @tusinahmed 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**